stc_game/Assets/Scripts/HUD/PotionCooldown.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PotionCooldown : MonoBehaviour
+{
+    [SerializeField] public Transform parent;

```

Comment:
this serializeField should be above the variable

Reply:
ok